x-pack/plugins/gis/public/actions/store_actions.js

```diff
@@ -116,6 +117,12 @@ export function mapReady() {
   };
 }
 
+export function mapDestroyed() {
```

Comment:
This and mapReady are pretty much the same function. We could just pass a boolean value to mapReady and reuse it in place of dispatching mapDestroyed